spec/en/reference.md

```diff
@@ -185,9 +215,32 @@ An agency may use the ridership.txt file to record aggregate level ridership cou
 | ridership_end_date | **Required** | The **ridership_end_date** field contains the date of the end of the rideship count. **ridership_end_date** may be the same date or later than **ridership_start_date**. |
 | ridership_start_time | Optional | The **ridership_start_time** field contains the time of the start of the rideship count on the date specified in **ridership_start_date**. The time format is HH:MM:SS. |
 | ridership_end_time | Optional | The **ridership_end_time** field contains the time of the end of the rideship count on the date specified in **ridership_end_date**. If **ridership_start_date** and **ridership_end_date** are the same, **ridership_end_time** must be later than **ridership_start_time**. |
+| service_id | Optional | The **service_id** field contains an ID that uniquely identifies a set of dates over which to aggregate ridership. This value is referenced from the [calendar.txt](https://github.com/google/transit/blob/master/gtfs/spec/en/reference.md#calendar.txt) file. The **ridership_start_date** to **ridership_end_date** date range must fully contain the date range specified for the associated **service_id** through the **start_date** and **end_date** fields of _calendar.txt_.  Use this field to indicate aggregation of ridership over day-of-the-week sets preexisting in GTFS service IDs. To aggregate over custom day-of-the-week sets within the **ridership_start_date** to **ridership_end_date** date range, use the following binary date selection fields.   |
```

Comment:
Hyperlink `[calendar.txt](https://github.com/google/transit/blob/master/gtfs/spec/en/reference.md#calendar.txt)` should be changed to `[calendar.txt](https://github.com/google/transit/blob/master/gtfs/spec/en/reference.md#calendartxt)`. Removes the `.` from `claendartxt` to locate correct anchor.